python/nano/tutorial/notebook/training/pytorch-lightning/accelerate_pytorch_lightning_training_multi_instance.ipynb

```diff
@@ -0,0 +1,260 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<style>\n",
+    "    .rst-content blockquote {\n",
+    "        margin-left: 0px;\n",
+    "    }\n",
+    "    \n",
+    "    blockquote > div {\n",
+    "        margin: 1.5625em auto;\n",
+    "        padding: 20px 15px 1px;\n",
+    "        border-left: 0.2rem solid rgb(59, 136, 219);  \n",
+    "        border-radius: 0.2rem;\n",
+    "        box-shadow: 0 0.2rem 0.5rem rgb(0 0 0 / 5%), 0 0 0.0625rem rgb(0 0 0 / 10%);\n",
+    "    }\n",
+    "</style>\n",
+    "\n",
+    "[View the runnable example on GitHub](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/training/pytorch-lightning/accelerate_pytorch_lightning_training_multi_instance.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accelerate PyTorch Lightning Training using Multiple Instances"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`bigdl.nano.pytorch.Trainer` API extends PyTorch Lightning Trainer with multiple integrated optimizations. You can instantiate a Nano `Trainer` with specified `num_processes` to benefit from multi-instance training on a server with multiple CPU cores or sockets, so that the workload can make full use of all CPU cores.\n"
```

Comment:
Not sure if this sentence is accurate “so that the workload can make full use of all CPU cores.”